recipes-containers/kube-router/files/launch-kube-router.sh

```diff
@@ -24,8 +24,13 @@ if [[ $? -ne 0 ]] || [[ $DEVICE_ID == null ]]; then
 fi
 
 POD_CIDR=EDGE_PODCIDR
+CONFIG="/run/kuberouter.conflist"
 
-exec env NODE_NAME=${DEVICE_ID} KUBE_ROUTER_CNI_CONF_FILE=EDGE_CNI_CONF/10-kuberouter.conflist EDGE_BIN/kube-router \
+if [[ ! -f ${CONFIG} ]]; then
+	cp EDGE_CNI_CONF/10-kuberouter.conflist ${CONFIG}
+fi
+
+exec env NODE_NAME=${DEVICE_ID} KUBE_ROUTER_CNI_CONF_FILE=${CONFIG} EDGE_BIN/kube-router \
 --v=1 \
```

Comment:
This one still have the verbose options on? Can we reduce logging, too?

Reply:
After discussion, we are leaving it as 1 for now as we first want to investigate the consequences of lowering it, this is a kubernetes/kube-router internal verbosity level and it may include useful debug information. 

FYI @sk-arm